importer/main.py

```diff
@@ -250,24 +245,17 @@ def location_extras(resource, payload_string):
         payload_string = json.dumps(obj, indent=4)
 
     try:
-        if physicalType == "building":
-            payload_string = payload_string.replace("$pt_code", "bu").replace(
-                "$pt_display", "Building"
-            )
-        elif physicalType == "jurisdiction":
-            payload_string = payload_string.replace("$pt_code", "jdn").replace(
-                "$pt_display", "Jurisdiction"
-            )
+        if len(physicalType.strip()) > 0:
+            payload_string = payload_string.replace("$pt_display", physicalType)
+        if len(physicalTypeCode.strip()) > 0:
+            payload_string = payload_string.replace("$pt_code", physicalTypeCode)
         else:
```

Comment:
This else would only happen if the physicalTypeCode is missing but not if the physicalType is missing
Wondering is that an ok assumption to make? That whoever is filling in the csv will always fill in for both in cases that need filling in?

Reply:
@Wambere  yes, the assumption is right.